earthaccess/results.py

```diff
@@ -325,7 +325,6 @@ def data_links(
             else:
                 # we are not in us-west-2, even cloud collections have HTTPS links
                 return https_links
-        return https_links
```

Comment:
This is just cosmetic (this line would never be called, so I decided to remove it)

Reply:
:+1: I tend to go the other way and drop the `else` statement, but six-of-one...